app/overrides/decidim/consultations/question_votes_controller.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+Decidim::Consultations::QuestionVotesController.class_eval do
```

Comment:
This is one approach, but I think we will need to override the whole file which makes me thing.
Why don't just create another route for delegation votes and in the overriden view just point the url of the form/ajax to that new route.
I think would be the more neat approach.